CHANGELOG.md

```diff
@@ -117,9 +117,9 @@ Pull requests:
 
 The focus state of components now meets the new WCAG 2.1 level AA requirements.
 
-You must update your component’s focus state to make your design consistent with our new focus styles.
+You must update your component’s focus state to make your design is consistent with our new focus styles.
```

Comment:
I think this 'is' might be a mistake.

Reply:
Oops, yes, sorry - I introduced this error. Will fix.

Reply:
Fixed in https://github.com/alphagov/govuk-frontend/pull/1513/commits/46b32a88c1b7da1af2ba697a405dec8a242431ca.